applications/GeoMechanicsApplication/custom_constitutive/linear_elastic_plane_strain_2D_law.h

```diff
@@ -69,22 +45,11 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoLinearElasticPlaneStrain2DLaw
     /// Counted pointer of LinearPlaneStrainK0Law
     KRATOS_CLASS_POINTER_DEFINITION( GeoLinearElasticPlaneStrain2DLaw );
 
-    ///@name Life Cycle
-    ///@{
-
     /**
      * @brief The clone operation
      */
```

Comment:
Only if you have to push other changes anyway (because it's really not important and not worth the 2 hour pipeline wait): but could you remove this trivial comment?